crates/autopilot/src/shadow.rs

```diff
@@ -201,6 +201,7 @@ impl RunLoop {
             self.score_cap,
             self.solve_deadline,
             self.fee_policy.clone(),
+            HashMap::new(),
```

Comment:
Would be cool if we could get the correct fee policies as part of the shadow competition as well but that is probably not straight forward since we don't have a DB connection in this case.

Reply:
correct

Reply:
@MartinquaXD do you maybe know why we don't have the access to db? for security reasons so we don't end up changing something?
I am worried that onboarding new solvers and checking the protocol fee mechanism will be imposible without db connection. OTOH, we do have staging if necesarry.

Reply:
I think when the `shadow` autopilot was written there was no need to store anything in the DB.
And over all I like that we don't need any DB connection for it to run.

Reply:
so do you think is it worth to modify the code of the shadow to be able to read the order and compute the fee policies or maybe is not worth the investment ?

Reply:
I think for testing fee policies we might have to add the original quote to the order (or some other indicator that fees should be applied) in `/auction`. However, **if** we decide to go that route we should do it in a separate PR.
With that taken care of the only reason the `shadow` autopilot would need a DB is to store the data we need before calling `/settle` which I think is outside of the scope of what the `shadow` environment was initially envisioned to do. Originally it was only supposed to be a test bed to onboard new solvers without risk.